api/handlers/containerhandler.py

```diff
@@ -154,6 +155,21 @@ def _filter_permissions(self, result, uid, site):
         if user_perm.get('access') != 'admin':
             result['permissions'] = [user_perm] if user_perm else []
 
+    def get_jobs(self, cont_name, cid):
+        try:
+            self.config = self.container_handler_configurations[cont_name]
+            self.storage = self.config['storage']
+            c = self._get_container(cid)
+            permchecker = self._get_permchecker(c)
+            result = permchecker(noop)('GET', cid)
+        except APIStorageException as e:
+            self.abort(400, e.message)
```

Comment:
I would remove the try block here. Permissions checking shouldn't be wrapped in a try block as it will self.abort with it's own error code and message, as will `_get_container` (but will error with an APIStorageException as this will here). A lookup in our configuration map is safe.